packages/apollo-react-relay-duct-tape/src/storeObservation/compiledHooks/useCompiledPaginationFragment.ts

```diff
@@ -128,7 +128,10 @@ function useLoadMore({
       // TODO: Measure if invoking `refetch` leads to React updates and if it
       //       makes sense to wrap it and the following setIsLoadingMore(true)
       //       call in a batchedUpdates callback.
-      const disposable = refetch(newVariables, refetchOptions);
+      const disposable = refetch(
+        newVariables,
+        refetchOptions as RefetchOptions
```

Comment:
Is this casting unavoidable?

Reply:
Perhaps if the refetching was extracted from useRefetchableFragment itself and shared between the two hooks. I’ll mull it over when I have some refactor time, but for now this is a fine-iSH private detail imo.